packages/shadcn/src/components/ui/slider.tsx

```diff
@@ -0,0 +1,25 @@
+'use client';
+
+import * as React from 'react';
+import * as SliderPrimitive from '@radix-ui/react-slider';
```

Comment:
Rather than use this, import each of the things you need directly from `react` and `@radix-ui/react-slider`